### DISCUSSION TITLE
Welcome to stability-sdk Discussions!

### DISCUSSION THREAD

**Discussion by dmarx** [Stability-AI/stability-sdk#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dmarx** [comment]
Hi! I'm David, aka [@DigThatData](https://twitter.com/DigThatData)! I'm one of the lead engineers on the API team.

**Comment by tashabaxter** [comment]
Howzit! Tasha here, aka (Cazimi on Discord) Head of Community management - people business :)
